maintainers/maintainer-list.nix

```diff
@@ -7143,6 +7143,12 @@
     githubId = 17120571;
     name = "Xinhao Luo";
   };
+  nichtsfrei = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add nichtsfrei`` and sort it alphabetically.

Reply:
Thank you.